blockchain.go

```diff
@@ -698,11 +727,6 @@ func (b *Blockchain) newFVMContextFromHeader(header *flowgo.Header) fvm.Context
 	return fvm.NewContextFromParent(
 		b.vmCtx,
 		fvm.WithBlockHeader(header),
-		fvm.WithReusableCadenceRuntimePool(
-			reusableRuntime.NewReusableCadenceRuntimePool(
-				1,
-				runtime.Config{Debugger: b.debugger, AccountLinkingEnabled: true}),
-		),
```

Comment:
Was this removal intentional? Is it not needed anymore?

Reply:
yeah, @m-peter pointed out[0] in coverage PR that it is not needed anymore as config will be passed via parent context.

[0] https://github.com/onflow/flow-emulator/pull/332#discussion_r1135265180

Reply:
Exactly, I observed that it was redundant for the `Debugger` & `AccountLinkingEnabled` config values. Moreover, the removal was necessary for the coverage reporting feature to work properly. However, at the time of writing that comment, the `ReusableCadenceRuntimePool` was responsible for also initializing the `CoverageReport` object. This was later changed. If there's no specific reason for constructing a new `ReusableCadenceRuntimePool`, then I would say that it's safe to remove.

Reply:
Sounds good, thanks for the explanation! I was just wondering how the removal relates to this PR

Reply:
ah yeah sorry, actually it was something left from when I tested if removing was safe, then it somehow came into this PR unintentionally, but then I thought adding would not make sense, for the sake of it. Sorry created a bit confusion